app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>StaticPages#help</h1>
+<p>Find me in app/views/static_pages/help.html.erb</p>
```

Comment:
plz, you should use `I18n`

Note: I18n gem support multi-language, format DateTime  and format currency for your project.

read link: https://guides.rubyonrails.org/i18n.html